build-support/scripts/proto-tools.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+unset CDPATH
+
+set -euo pipefail
+
+readonly PROTOC_GEN_GO_VERSION="$(grep github.com/golang/protobuf go.mod | awk '{print $2}')"
+readonly MOG_VERSION='v0.2.0'
```

Comment:
TODO: figure out how to easily opt out of the automation when editing mog itself

Reply:
Figured it out.